docs/en_US/CommunitySharings/HpoComparision.md

```diff
@@ -98,6 +98,7 @@ The total search space is 1,204,224, we set the number of maximum trial to 1000.
 | HyperBand     |0.414065|0.415222|0.417628|
 | HyperBand     |0.416807|0.417549|0.418828|
 | HyperBand     |0.415550|0.415977|0.417186|
+| GP            |0.414353|0.418563|0.420263|
```

Comment:
seems that there's many hyper-params in GP tuner, pls consider adding experiments for them.

Reply:
Also please put three times result here.

Reply:
Conf with full hyper-parameter are added in test-config. 
Two more times results are added.